src/ui/public/styles/variables/bootstrap-mods.less

```diff
@@ -1,52 +1,18 @@
-//== Colors
```

Comment:
Wait, so this is an actual bootstrap file? Not one we've just added our modifications to?